doc/move_group_interface/src/move_group_interface_tutorial.cpp

```diff
@@ -201,7 +201,10 @@ int main(int argc, char** argv)
 
   // Now, let's modify one of the joints, plan to the new joint space goal and visualize the plan.
   joint_group_positions[0] = -1.0;  // radians
-  move_group.setJointValueTarget(joint_group_positions);
+  bool within_bounds = move_group.setJointValueTarget(joint_group_positions);
+  if(!within_bounds) {
+    RCLCPP_WARN(LOGGER, "Target joint position(s) were outside of limits, but we will plan and clamp to the limits ");
```

Comment:
This warning (debug?) could be useful in MGI as well.

Reply:
Maybe the return value from this function should just be marked [[nodiscard]]?